frame/system/src/lib.rs

```diff
@@ -937,7 +937,13 @@ impl<T: Trait> Module<T> {
 	pub fn deposit_event_indexed(topics: &[T::Hash], event: T::Event) {
 		let block_number = Self::block_number();
 		// Don't populate events on genesis.
-		if block_number.is_zero() { return }
+		if block_number.is_zero() {
+			// Print a message only in tests warning users about this behavior.
+			#[cfg(test)]
```

Comment:
This does not changes anything outside the crate. The test feature is only enabled for a crate when the crate itself is build for testing. This means, any other crate using `frame_system` will not print anything.

Reply:
Hmm, is there some attribute that does what I intend?

Reply:
Nope^^